docs/user-guide/layer-settings.md

```diff
@@ -92,25 +92,25 @@ In particular, the user is allowed to:
 
 ## Fields
 
-From this section of the *Settings* panel, [Mapstore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows the user to add aliases to layer fields.
+From this section of the *Settings* panel, [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows the user to add aliases to layer fields.
 
 <img src="../img/layer-settings/fields_settings.jpg" class="ms-docimage"  style="max-width:500px;"/>
 
-The panel lists the fields that make up the layer. For each field the following is highlighted:
+The panel shows the fields (feature attributes) of the layer: For each field the following are specified:
```

Comment:
The panel shows the fields (feature attributes) of the layer. For each field the following are specified: